.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -340,6 +340,8 @@ jobs:
               -DCMAKE_BUILD_TYPE=Release \
               -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=$GITHUB_WORKSPACE/runtime-build/lib \
               -DPYTHON_EXECUTABLE=$(which python${{ matrix.python_version }}) \
+              -DPython_ROOT_DIR=$(python${{ matrix.python_version }} -c "import sys; print(sys.prefix)") \
+              -DPYTHON_VERSION_TO_FIND=${{ matrix.python_version }} \
```

Comment:
Is the version necessary if you already specify the path?

Reply:
That's a good question; I put both because I was concerned that there may be multiple Python versions installed under the same root directory that we pass in as `Python_ROOT_DIR`. I'm not sure if it's strictly necessary, but it puts my mind at ease! :sweat_smile: (Who knows how the Python installations on our runners will change in the future).

It's unfortunate, nanobind requires that you have a `find_package(Python)` command in your CMake config, otherwise you get an error

```
CMake Error at /.../lib/python3.12/site-packages/nanobind/cmake/nanobind-config.cmake:4 (message):
  You must invoke 'find_package(Python COMPONENTS Interpreter Development
  REQUIRED)' prior to including nanobind.
```

and you have to be very explicit in the `find_package` command to make sure that it find the right Python interpreter. I'd be curious to know if there is a more concise way to do this, but at least we know that it works now as is.

Reply:
Yeah that's fair, I also remember struggling with the `find_python` command in some instances 😬